ReactAndroid/src/main/jni/react/jni/FileRAMBundle.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <memory>
+
+#include <android/asset_manager.h>
+#include <cxxreact/RAMBundle.h>
+
+namespace facebook {
+namespace react {
+
+class FileRAMBundle : public RAMBundle {
+  /**
+   * This implementation reads modules as single file from the assets of an apk.
+   */
+ public:
+  FileRAMBundle() = default;
+  FileRAMBundle(
+      AAssetManager *assetManager,
+      const std::string &moduleDirectory,
+      std::unique_ptr<JSBigString> startupScript);
+  ~FileRAMBundle() {}
+
+  static bool isFileRAMBundle(
+      AAssetManager *assetManager,
+      const std::string &assetName);
+
+  std::string getSourcePath() const override;
+  std::string getSourceURL() const override;
+  std::shared_ptr<const JSBigString> getStartupScript() const override;
+  Module getModule(uint32_t moduleId) const override;
+
+  BundleType bundleType = BundleType::FileRAMBundle;
```

Comment:
```suggestion
  BundleType getBundleType() const override;
```
Check: https://github.com/callstack/react-native/commit/103467bc5985d669d476241e94731c8690108a52